README.md

```diff
@@ -14,13 +14,25 @@ CMake version 3.12 is required to use these libraries.
 ### Development Environment
 Project contains files to work on Windows or Linux based OS.
 
+> Note: Project contains git submodules required to build. Before building run `git submodule update --init --recursive`
+
 #### Windows
 Use PowerShell to run {projectDir}/build.ps1
 
 #### Linux
-Use bash to run {projectDir}/build-tools/build.sh
-
-Output libraries are created in /build folder
+- Install `libssl`

```

Comment:
specify version please (or say it doesn't matter) not everyone is on ubuntu and this will get out of date over time.

I'd prefer saying "install openssl version .... On ubuntu XX.XX this is libssl-dev"

same for libuuid.

Reply:
makes sense.  Will update it. tks